gap/semigroups/semifp.gi

```diff
@@ -541,3 +541,100 @@ IsCollsElms, [IsFpSemigroup, IsElementOfFpSemigroup],
 function(S, x)
   return SEMIGROUPS.ExtRepObjToWord(ExtRepOfObj(x));
 end);
+
+InstallMethod(IsomorphismFpSemigroup,
+"for an inverse partial perm semigroup",
+[IsPartialPermSemigroup and IsInverseActingSemigroupRep],
+function(M)
+  local add_to_odd_positions, S, SS, G, GG, s, g, F, rels, fam, alpha,
+  beta, lhs, rhs, map, H, o, comp, U, rhs_list, conj, MF, T, inv, rel, x, y, m,
+  i;
+
+  if not IsFactorisableInverseMonoid(M) then
+    TryNextMethod();
+  fi;
+
+  add_to_odd_positions := function(list, s)
+    local i;
+    for i in [1, 3 .. Length(list) - 1] do
+      list[i] := list[i] + s;
+    od;
+    return list;
+  end;
```

Comment:
Couldn't this whole method just be `list{[1, 3 .. Length(list) - 1]} := list{[1, 3 .. Length(list) - 1]} + s`?

Reply:
Yes, changed.